common/COMMON_moveReportToRSlims.py

```diff
@@ -1,59 +1,65 @@
 """Python interface to GenoLogics LIMS via its REST API.
 
-If a PDF is attached to an artifact in a step, this PDF is downloaded and put in 
-a shared file surface with LIMS RS. 
-The name of the file should be the same as in Clarity LIMS where the name is set when the file is created.
-The file must start with sampleName for LIMS RS to match it to the right sample.
+Description: If a PDF is attached to an artifact in a step, this PDF is downloaded and put in a shared file surface with LIMS RS. The name of the file should be the same as in Clarity LIMS where the name is set when the file is created. The file must start with sampleName for LIMS RS to match it to the right sample.
 
-Script Classification: Non critical - The reports can be manually downloaded and uploaded to LIMS RS 
+Script Classification: Not critical - The reports can be manually downloaded and uploaded to LIMS RS 
 """
-from genologics.lims import *
-from genologics.entities import *
-import sys
-import getopt
+from genologics.entities import Process
+import sys, os
+import logging
+
+# Import from this repository
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "customPackageSMD"))
+from py3functions import setUpArgs, setUpLIMS, set_logger_log_start_script, logger_end_script
+
+RS_LIMS_PAHT = "/mnt/limsrs-clarity/PDF/in/"
```

Comment:
```suggestion
RS_LIMS_PATH = "/mnt/limsrs-clarity/PDF/in/"
```

typo. remember to change elsewhere.